src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -124,16 +127,10 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
             sortBy:(d:Mutation[])=>MutantCopiesColumnFormatter.getDisplayValueAsString(d, this.props.sampleIdToClinicalDataMap, this.getSamples())        
         };
 
-        this._columns[MutationTableColumnType.COPY_NUM] = {
-            name: "Copy #",
-            tooltip: (<span>I just want something that works</span>),
-            render:(d:Mutation[])=>PatientDiscreteCNAColumnFormatter.renderFunction(d,
-                        this.props.molecularProfileIdToMolecularProfile as {[molecularProfileId:string]:MolecularProfile},
-                        this.props.discreteCNACache as DiscreteCNACache, this.props.sampleIdToClinicalDataMap, this.getSamples(), this.props.sampleManager),
-            sortBy:(d:Mutation[])=>DiscreteCNAColumnFormatter.getSortValue(d, this.props.molecularProfileIdToMolecularProfile as {[molecularProfileId:string]:MolecularProfile},
-                        this.props.discreteCNACache as DiscreteCNACache, this.props.sampleIdToClinicalDataMap, this.getSamples()),
-            filter:(d:Mutation[], filterString:string)=>DiscreteCNAColumnFormatter.filter(d, this.props.molecularProfileIdToMolecularProfile as {[molecularProfileId:string]:MolecularProfile},
```

Comment:
I removed the filtering since the display values are just icons (displaying total copy number) - filtering by what's displayed like `5` wouldn't really be helpful